cloudsync_onedrive.py

```diff
@@ -95,6 +99,10 @@ class ErrorCode:
     Malformed = "malformed"
 
 
+class OneDriveResourceModifiedError(CloudTemporaryError):
```

Comment:
The rest of the MR looks good, just move this up into cloudsync, and we're good to go

Reply:
done.